apps/web/src/app/(app)/(auth)/reset/page.tsx

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import ResetForm from "@/components/auth/reset-form";
+
+import { Card, CardContent, CardHeader } from "@energyleaf/ui";
+
+export const runtime = "nodejs";
+
+export default function Page() {
+    return (
+        <Card className="w-full max-w-xl">
+            <CardHeader>
+                <div className="flex flex-row items-center justify-center gap-2">
+                    <Image
+                        alt="Energyleaf Logo"
+                        className="h-16 w-16"
+                        height={499}
+                        src="/image/logo/logo.png"
+                        width={499}
+                    />
+                    <h1 className="text-4xl">Energyleaf</h1>
+                </div>
+            </CardHeader>
+            <CardContent>
+                <ResetForm />
```

Comment:
Ich bin mir nicht sicher, ob man das irgendwie hier leicht so umsetzen kann, dass man die Klasse nicht bis auf diese Stelle mit der forgot Form duplikaten muss.

Aber ich wollte darauf mal hinweisen, vllt hat da ja wer Ahnung von

Reply:
Man kann die ganze "Card" die für den Signin/Signup verantwortlich ist, in eine eigene Komponente auslagern. Und dann den Inhalt, in dem Fall die ResetForm einfügen.
Für Titel oder Beschreibungen könnte man extra Attribute anlegen.

```typescript
interface Props {
    children: React.ReactNode
    description: React.ReactNode
    ....
};

export default function OnboardingCard({ children, description, ...}: Props) {
    return (
        <Card>
            ....
            <CardContent>
                {children}
            </CardContent>
        </Card>
    );
}